### PR TITLE
[12.0] [MIG] account_asset_batch_compute

### PR DESCRIPTION
**Pull request by etobella**
No major changes

@JordiBForgeFlow

### CODE REVIEW

**Comment by jarroyomorales** [comment]
Add `queue` to `oca_requirements.txt`

**Review by jarroyomorales — APPROVED** [review]
LGTM! :+1:

**Review by sbejaoui — APPROVED** [review]
Very useful thanks.

It's sad that this module exists since v8 only in PRs. 

https://github.com/OCA/account-financial-tools/pulls?q=is%3Apr+is%3Aopen+account_asset_batch_compute

File: account_asset_batch_compute/tests/test_account_asset_batch_compute.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+            'method_number': 1,
+            'method_period': 'month',
+            'prorata': False,
+        })
+        today = date.today()
+        first_day_of_month = date(today.year, today.month, 1)
+        self.nextmonth =\
+            first_day_of_month + relativedelta.relativedelta(months=1)
+        self.nextmonth = first_day_of_month + relativedelta.relativedelta(
+            months=1)
```
Comment (sergiocorato) [inline account_asset_batch_compute/tests/test_account_asset_batch_compute.py:60]: duplicated code?

File: account_asset_batch_compute/wizards/account_asset_compute_views.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<openerp>
+<data>
```
Comment (sergiocorato) [inline account_asset_batch_compute/wizards/account_asset_compute_views.xml]:
```suggestion
<odoo>
```

File: account_asset_batch_compute/wizards/account_asset_compute_views.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        <field name="inherit_id" ref="account_asset_management.account_asset_compute_view_form"/>
+        <field name="arch" type="xml">
+            <field name="date_end" position="after">
+                <field name="batch_processing" />
+            </field>
+        </field>
+    </record>
+
+</data>
+</openerp>
```
Comment (sergiocorato) [inline account_asset_batch_compute/wizards/account_asset_compute_views.xml]:
```suggestion
</odoo>
```

File: account_asset_batch_compute/tests/test_account_asset_batch_compute.py [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+from datetime import date
+from dateutil import relativedelta
+
+from odoo.addons.queue_job.job import Job
+
+DELAY2 = ('odoo.addons.account_asset_batch_compute.wizards.'
+          'account_asset_compute.async_asset_compute')
+DELAY1 = ('odoo.addons.account_asset_batch_compute.models.'
+          'account_asset.async_compute_entries')
+
```
Comment (sergiocorato) [inline account_asset_batch_compute/tests/test_account_asset_batch_compute.py]: What does this code do?

**Comment by sbejaoui** [comment]
Hi @etobella ,

I made a PR to your branch to resolve @sergiocorato comments. https://github.com/tegin/account-financial-tools/pull/1

**Comment by etobella** [comment]
Thanks @sbejaoui @sergiocorato

**Comment by etobella** [comment]
@sergiocorato I think all your comments have been attended.

**Comment by jarroyomorales** [comment]
ping @sergiocorato

**Pull request merged**